core-initrd/latest/bin/ubuntu-core-initramfs

```diff
@@ -668,13 +672,32 @@ def verify_missing_dlopen(destdir, libdir):
 
     return not fatal
 
+
 def get_deb_arch(sysroot):
     proc_env = os.environ.copy()
     proc_env["DPKG_DATADIR"] = sysroot + "/usr/share/dpkg"
     out = check_output(["dpkg-architecture", "-q",
                         "DEB_HOST_MULTIARCH"], env=proc_env).decode("utf-8")
     return out.splitlines()[0]
 
+
+def get_ubuntu_release(sysroot):
+    with open(os.path.join(sysroot, "etc/os-release"), "r") as f:
+        data = f.read()
+        regex = re.compile('^VERSION_ID="(.*)"$')
+        for line in data.splitlines():
+            res = regex.search(line)
```

Comment:
Do we need to do it for every line? Cannot we search in the whole data?

Reply:
Done